openfisca_core/parameters.py

```diff
@@ -36,7 +37,7 @@
 # 'unit' and 'reference' are only listed here for backward compatibility.
 #  It is now recommended to include them in metadata, until a common consensus emerges.
 COMMON_KEYS = {'description', 'metadata', 'unit', 'reference', 'documentation'}
-ALLOWED_PARAM_TYPES = (float, int, bool, type(None))
+ALLOWED_PARAM_TYPES = (float, int, bool, type(None), typing.List)
```

Comment:
Any preference between ```typing.List``` and just ```List```? Asking just for consistency (current usage is ```from typing import XXX```, but that is not consistent with other libraries usage, for example ```numpy``` and so on).

Reply:
No preference.

Reply:
Could you use ```from typing import List``` just for consistency? Otherwise GTM :)